### ISSUE TITLE
Bring back VDAVideoDecoder (DXVA via D3D9) for old GPUs

### ISSUE THREAD

**Issue by FK2FAGHMS** [Alex313031/thorium-legacy#114]
**Is your feature request related to a problem? Please describe.**
Prior to version 110, it's possible to enable GPU video acceleration on old GPUs by using the command line --disable-gpu-driver-bug-workarounds and flags like "Override software rendering". When you play a video (h264), go to chrome://media-internals and it shows that it uses VDAVideoDecoder:
![image](https://github.com/user-attachments/assets/0c27a34b-a307-418b-97ee-eb92702a37be)

If you use any of backported browsers (Supermium, Thorium based on Supermium and so on), it will show you this:
![image](https://github.com/user-attachments/assets/85b27e68-0501-4118-94be-5fc3d73da9f1)
![image](https://github.com/user-attachments/assets/dd36f981-3860-4c72-8349-30fb786acb10)
Looks like they just removed/disabled VDAVideoDecoder with version 110 since it's not targeted at Windows 7 anymore.

**Describe the solution you'd like, including relevant patches or source**
If you'd be able to bring back VDAVideoDecoder and DXVA with DirectX9, you would:
Possibly enable GPU video acceleration on Windows XP
Enable at least 15 years old GPUs to be able to use their video decoding functionality
Simply, you would become a legend for whose use/have to use a so old PC for obvious reasons.

**Additional Notes**
To test it yourself, you can either try to find a laptop with Geforce 7-8-9-200-300 series card or find its desktop variant and a motherboard
supports it (IDK if they are compatible with newest motherboards).
**I would happily test as well when I have the time for it.**

**Comment by r2r2r2o** [comment]
I was able to utilize VDAVideoDecoder with Chromium 115.0.5790.171 (+ enhanced-h264ify) on Windows 10 on DirectX 10.1 GPU (RV620 - HD3470). Nevertheless, it should be possible also on Windows 7. The flags set is: 

`--ignore-gpu-blocklist --enable-features=DXVAVideoDecoding --disable_d3d11_video_decoder --use-angle=d3d9`

**Comment by Half-Modern** [comment]
Yeah, that issue was outdated actually. I already knew that 115 was the last version to include it behind a command line but it's been months since I stopped using that laptop with DX10. It'll stay open though as it might help someone else.
